### PR TITLE
Update prices of mobile apps

### PR DESCRIPTION
**Pull request by fbartels**
Hi, I've noticed that the prices on the page do not match the prices in the app stores. This pr adjusts the prices on the page.

### CODE REVIEW

**Comment by kevinpapst** [comment]
AFAIK this depends on the tax rates of your accounts country and therefor can only be shown without tax here - see screenshot: `excl. VAT`:
Bildschirmfoto 2019-12-30 um 16 22 53
ping @hmr-it-jr

**Comment by fbartels** [comment]
Oh yes, VAT is a good point. I did not consider this.

Then the price for the iOS app can stay like it is, but the Android app still needs to be updated. I'll update the PR in a moment.

**Comment by hmr-it-jr** [comment]
The app will be free soon since I move my Codebase completely to ReactNative. Pro features will be available with In-App-Purchases. I just didn't want to adjust the prices too often.

The price for Android is: 1,90 € without vat.
The price for iOS is: 1,92 € without vat.

In the store I set the prices without vat so the specific vat of the current country is used.

File: _store/hmr-it-android-offline-tracker.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ title: Kimai2 Android offline tracker
 name: Kimai2 Android app for offline tracking
 intro: "Kimai2 Android app for offline tracking"
 developer: hmr-it
-date: "2019-08-20 08:10:00"
+date: "2019-13-30 16:30:00"
```
Comment (kevinpapst) [inline _store/hmr-it-android-offline-tracker.md]: I don't think this is a valid date ^^ Can you just revert that line please?

Comment (fbartels) [reply _store/hmr-it-android-offline-tracker.md]: Oh, indeed. one month too much.

Reverted to original date

**Pull request merged**